### ISSUE TITLE
508-defect-2: Plot or interment announcement should split into separate pages or use a heading

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#43343]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

- As a sighted user, I want groupings of fields to be visually clear and understandable
- As a screen reader user, I expect headings to be written as headings on the page to help me understand the hierarchy of a page and when a new section begins
- As a user with cognitive disabilities, I expect more context or hints to help me understand how to answer questions

## Details

The plot or internment allowance page attempts to cover three groups of questions onto the same page including:
- Where the Veteran was buried
- If they were buried in a national, federal, or state cemetary
- If the federal government contributed to the burial and how much was contributed

This UX results in a couple of issues including:
- Using a `fieldset` instead of a `heading` to provide extra content to each of the groups and fields (this doesn't work on some screen readers due to nested fieldsets)
- Absence of hint or help text (e.g., would users know if the Veteran's burial place was state, national, or federally owned?) 
- Deceptive groupings as the solid blue line falsely makes it seem as though all four questions are conditional against one another (when they're actually two separate groups)

## Acceptance Criteria

TBD

## Steps to Recreate

1. [Go to step 4 for plot or internment allowance](https://www.va.gov/burials-and-memorials/application/530/benefits/plot-allowance)
2. Answer no and yes for the questions
3. Confirm the blue line connects the two separate groupings
4. Inspect "Plot or internment allowance"
5. Confirm it's a legend and not a heading

## Proposed Solution (if known)

Short term, we may want to consider the following:
- Changing the legend into a heading and removing the outermost fieldset
- Adding spacing between the two conditional groups or subheadings to split them apart

Long term, we may want to consider splitting up the groups onto separate pages ([one thing per page](https://www.smashingmagazine.com/2017/05/better-form-design-one-thing-per-page/)).

## Screenshots or Trace Logs
<img width="744" alt="Screen Shot 2022-06-24 at 11 14 27 AM" src="https://user-images.githubusercontent.com/14154792/175567963-af5f1005-753b-441c-bf1b-ea79f4763d10.png">

**Comment by fiorella-io** [comment]
With the new Burial redesign, we've split these questions into one question per page and [updated the logic (check step 4)](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1691512530884/67e3bc6677c8d17cf6fc8848319a6e40ebced1af?wid=0-1694615398728). 

Designs:
- [4.4](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/8VA1MMO)
- [4.5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/MmGZJJa)
- [4.6](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/a/WdgA00w)
